tests/src/Functional/ContentPersonRenderTest.php

```diff
@@ -0,0 +1,280 @@
+      'Additional information',
+      $content->text()
+    );
+
+    // Assert documents are not present.
+    $this->assertStringNotContainsString(
+      'Related documents',
+      $content->text()
+    );
+
+    // Test person with all fields.
+    $node_with_all_fields = $this->createExamplePersonWithAllFields();
+    $this->drupalGet($node_with_all_fields->toUrl());
+    $crawler = $client->getCrawler();
+
+    $content_banner = $crawler->filter('.bcl-content-banner');
+
+    $this->assertEquals(
+      'Stefan Mayer',
+      $content_banner->filter('.card-title')->text()
+    );
+    $this->assertEquals(
+      'DG Test',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(1)')->text()
+    );
+    $this->assertEquals(
+      'Director',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(2)')->text()
+    );
+    $this->assertEquals(
+      'Germany',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(3)')->text()
+    );
+
+    // Assert content banner image.
+    $image = $content_banner->filter('img');
+    $this->assertCount(1, $image);
+    $this->assertCount(1, $image->filter('.card-img-top'));
+    $this->assertStringContainsString(
+      'image-test.png',
+      $image->attr('src')
+    );
+    $this->assertEquals('Starter Image test alt',
+      $image->attr('alt')
+    );
+
+    $this->assertEquals(
+      'This field is used to add a short biography of the person.',
+      $content_banner->filter('p')->text()
+    );
+
+    $this->assertEquals(
+      'Twitter profile',
+      $content_banner->filter('div.mb-2 > div.mb-3 > a.standalone')->text()
+    );
+
+    $content = $crawler->filter('div.col-12.col-lg-10.col-xl-9.col-xxl-8');
+
+    $this->assertEquals(
+      'Additional information',
+      $content->filter('h2')->eq(0)->text()
+    );
+    $this->assertEquals(
+      'Additional information example field.',
+      $content->filter('div > p')->text()
+    );
+
+    $this->assertEquals(
+      'Related documents',
+      $content->filter('h2')->eq(1)->text()
+    );
+    $this->assertEquals(
+      'Event document test',
+      $content->filter('.bcl-file p')->text()
+    );
+  }
+
+  /**
+   * Tests the person rendered in 'Teaser' view mode.
+   */
+  public function testPersonRenderingTeaser(): void {
+    $node = $this->createExamplePersonWithAllFields();
+    // Build node teaser view.
+    $builder = \Drupal::entityTypeManager()->getViewBuilder('node');
+    $build = $builder->view($node, 'teaser');
+    $render = $this->container->get('renderer')->renderRoot($build);
+    $crawler = new Crawler((string) $render);
+
+    $article = $crawler->filter('article');
+    $this->assertCount(1, $article);
+
+    $this->assertEquals(
+      'Stefan Mayer',
+      $article->filter('h1.card-title')->text()
+    );
+    $image = $article->filter('img');
+    $this->assertCount(1, $image);
+    $this->assertCount(1, $image->filter('.card-img-top'));
```

Comment:
```suggestion
    $image = $article->filter('img.card-img-top');
    $this->assertCount(1, $image);
```
Unless you want to verify that we have no unexpected images beyond the 1st - but then we'd do it for the entire `$crawler` content.